sdk/spring/spring-integration-azure-core/src/test/java/com/azure/spring/integration/handler/MessageHandlerTest.java

```diff
@@ -1,130 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.spring.integration.handler;
```

Comment:
Why is this deleted instead of renaming?

Reply:
I did refactoring, and sometimes it just happened.

Reply:
Do you think it's possible for you to bring the git history back?